### DISCUSSION TITLE
Zoom link for remote sessions

### DISCUSSION THREAD

**Discussion by BeaLeydier** [gui2de/ppol768-spring23#121] [Announcements]
Hi team

As a reminder, weeks 4 and 5 of the syllabus will be taught remotely by Ali (the next in person session is on 02/23). We will also be broadcasting the zoom into the regular classroom in Reiss 112, so any of you that are on campus and prefer the classroom experience are more than welcome to come to the classroom directly; or you can join from your own laptop if that's your preference.

The zoom link for all four sessions will be : https://georgetown.zoom.us/my/gui2de

Thanks all!

**Comment by BeaLeydier** [comment]
Reminder @gui2de/ppol768-spring23, today will also be remote at the link above. Like on Tuesday, we will be broadcasting it in Reiss 112 and I will be present to answer office hour questions, so you are all welcome to join from home or from the classroom.

**Reply by cmg340** [reply]
Hi Bea! would it be possible to zoom in for office hours as well?

**Reply by BeaLeydier** [reply]
Yes, feel free to join the class link for it, I'll be on at 2:30pm!

**Comment by hamzamajoka** [comment]
Reminder: My office hours are from 3 pm to 5 pm today. You can use the following zoom link: https://georgetown.zoom.us/my/gui2de
